src/keyboard/textinput.js

```diff
@@ -25,6 +25,7 @@ var TextInput = function(parentNode, host) {
     text.setAttribute("autocorrect", "off");
     text.setAttribute("autocapitalize", "off");
     text.setAttribute("spellcheck", false);
+    text.setAttribute("aria-multiline", true);
```

Comment:
[`aria-multiline`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiline) is just to indicate that pressing `Enter` inserts a new line instead of submitting a form, but `<textarea>` has that semantic by default so it's not really adding anything here, I'll just remove it.